src/components/Products.jsx

```diff
@@ -26,12 +34,28 @@ const Products = () => {
           <Item.Header>{product.name}</Item.Header>
           <Item.Meta> {`${product.price}kr`}</Item.Meta>
           <Item.Description>{product.description}</Item.Description>
+          <button
+            data-cy="order-button"
+            onClick={() => addToOrder(product.id)}
```

Comment:
Does the `onClick` event need to be a callback here?

Reply:
For curious readers: We experimented together, and yes, the `() =>` is necessary.